components/eamxx/src/scream_config.h.in

```diff
@@ -45,8 +45,10 @@
 // Whether or not to run RRTMGP debug checks
 #cmakedefine SCREAM_RRTMGP_DEBUG
 
-// Whether monolithic kernels are on
+// Whether or not small kernels are on
```

Comment:
We should prob be more specific. This could be OFF, and yet p3 may use small kernels. So maybe I would rephrase as "Whether or not small kernels are used in ALL targets that support them".